cassandra/cluster.py

```diff
@@ -4244,7 +4244,14 @@ def one(self):
         you know a query returns a single row. Consider using an iterator if the
         ResultSet contains more than one row.
         """
-        return self._current_rows[0] if self._current_rows else None
+        row = None
+        if self._current_rows:
+            try:
+                row = self._current_rows[0]
+            except Exception:  # generator object is not subscriptable, PYTHON-1026
```

Comment:
```suggestion
            except TypeError:  # generator object is not subscriptable, PYTHON-1026
```

No reason not to specify IMO

Reply:
totally... no idea why I put Exception there.